examples/plot_quickstart_selection_rate.py

```diff
@@ -6,20 +6,23 @@
 Selection rates in census dataset
```

Comment:
The filename and the title of the notebook no longer match the content. Either rename the file and change title, or possibly just include the generation of this image in the other notebook.

Also--the exported image gets chopped for me, so it looks like this:
<img width="543" alt="image" src="https://github.com/fairlearn/fairlearn/assets/20565444/75cf41e2-1f6d-455c-984a-9382d40bbb24">

Reply:
Is it possible that the directory is incorrect? Right now, it's looking in auto_examples, but the images are in build/html/_images

Reply:
you might be right. do you want to try to change it?

Reply:
i'm not sure why it's not a problem with the current quickstart though... we'll see once it's merged.

Reply:
> you might be right. do you want to try to change it?

I can try to pass the examples directory instead - would be more straightforward to update the images there. But yeah, I am also not sure why it's not updating when we push updates. It looks like the auto-examples folder is hidden and untracked by GitHub, so there may be something I am missing with my repository setup on my local machine.

Reply:
auto_examples is generated from examples, so that's why it's not checked into the repo. It's meant to show up when you run the doc build.